doc/content/microreactors/gcmr/GCMR_Core_Neutronics.md

```diff
@@ -0,0 +1,100 @@
+
+The core comprises three types of fuel assemblies: Assembly A in the inner region, Assembly B in the middle, and Assembly C in the outer core region. Each fuel assembly incorporates TRISO fuel blocks containing 19.75 wt% of LEU fuel and Yttrium hydride moderator pins encased in FeCrAl envelopes. Additionally, burnable poison blocks, composed of Gd2O3 particles with a 25% packing fraction distributed axially, and Helium coolant channels are integrated. Assembly A's detailed design is provided in [Fig_3], while [Fig_4] illustrates the design differences among the three assemblies. Notably, Assemblies B and C share similarities, except for Assembly B's inclusion of a core center location for a control rod. Key design parameters of the GCMR core are:
+
+!media media/gcmr/core/Fig_3.png
+      style=display: block;margin-left:auto;margin-right:auto;width:80%;
+      id=Fig_3
+      caption= GCMR assembly of Type A
+
+!media media/gcmr/core/Fig_4.png
+      style=display: block;margin-left:auto;margin-right:auto;width:80%;
+      id=Fig_4
+      caption= Design of the three types of fuel assemblies in the core
+
+| Parameter (unit)| Value |
+| - | - |
+| Reactor Power (MWth) | 20.0 |
+| Core diameter (cm) | 242.0 |
+| Core height (cm) | 240.0 |
+| Active height (cm) | 200.0 |
+| Different core zones | 3 |
+| Number of control drums | 12 |
+| Lattice pitch (cm) | 20.8 |
+| Pin pitch (cm) | 2.0 |
+| TRISO fuel compact radius (cm) | 0.85 |
+| Moderator compact radius (cm) | 0.75 |
+| Cr coating thickness (cm) | 0.007 |
+| FeCrAl envelope thickness (cm)| 0.05 |
+| Burnable poison compact radius (cm) | 0.25 |
+| Coolant compact radius (cm) | 0.6 |
+| Control compact radius (cm) | 0.95 |
+| Fuel | TRISO, 40% packing fraction |
+| Coolant | He |
+| Moderator (Coating, Envelope)| YH1.8 (Cr, FeCrAl) |
+| Burnable poison absorber | Gd2O3 particles, 25% packing fraction |
+| Control rod | B4C (96% B-10 enrichment) |
+
+## Mesh
+
+MOOSE's Reactor Module was applied to create the mesh structure for the entire core of the GC-MR reactor. This tool was particularly useful in addressing intricate modeling prerequisites for the microreactor core such as fuel assemblies exhibiting different designs, distinct axial loading configurations of the burnable absorbers within the core, and modeling intricacies of control drum designs. The transition of the mesh from a 2D to a 3D structure, along with the incorporation of distinct axial regions, was made notably straightforward by employing the FancyExtruderGenerator. The tool's adaptability became especially useful when altering the mesh element size to examine the trade-off between accuracy and computational cost through coarser mesh. The meshing tool computational demands remained reasonable, making it a cost-effective choice for mesh generation, given the available mesh generation resources. [Fig_5]  illustrates a depiction of the 3-D GCMR, detailing both the axial and radial discretization.
```

Comment:
Reference reactor module

@article{shemon2023reactor,
 author = {Emily Shemon and Yinbin Miao and Shikhar Kumar and Kun Mo and Yeon Sang Jung and Aaron Oaks and Scott Richards and Guillaume Giudicelli and Logan Harbour and Roy Stogner},
  title = {MOOSE Reactor Module: An Open-Source Capability for Meshing Nuclear Reactor Geometries},
journal = {Nuclear Science and Engineering},
 volume = {0},
 number = {0},
  pages = {1-25},
  year  = {2023},
    doi = {10.1080/00295639.2022.2149231},
    URL = {https://doi.org/10.1080/00295639.2022.2149231},
 eprint = {https://doi.org/10.1080/00295639.2022.2149231}
}

Reply:
let s add that to the VTB bib file if it s not in there already:?

Reply:
Added to VTB bib